ts/annotations/annotations.src.ts

```diff
@@ -1608,6 +1645,27 @@ extend(chartProto, /** @lends Highcharts.Chart# */ {
         this.annotations.forEach(function (annotation): void {
             annotation.redraw();
         });
+        this.delayRendering(); // #12584
+    },
+
+    delayRendering: function (this: Highcharts.AnnotationChart): void {
+        var chart = this;
+
+        chart.annotations.forEach(function (annotation): void {
+            var defer = annotation.options.defer,
+                seriesAnimDuration = animObject(defer).duration,
+                fadeInDuration = seriesAnimDuration ? Math.min(seriesAnimDuration, 200) : 0,
+                annotationsGroup = [annotation.shapesGroup, annotation.labelsGroup];
```

Comment:
We should simply use `annotation.graphic` which is the container for  `shapesGroup` and `labelsGroup`